sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -29,8 +29,7 @@ and includes six main functions:
 </dependency>
```

Comment:
should we take advantage of this PR and update the version?

Reply:
Yap. I can do it in this PR